### PR TITLE
fix: truncated order status

### PR DESCRIPTION
**Pull request by anhanh11001**
Fixes: #2073 

Screenshots for the change:

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Make the label have a little more padding and rounded corder please

**Comment by iamareebjamal** [comment]
See 1 ticket -> See 1 ticket(s)

**Comment by iamareebjamal** [comment]
> Make the label have a little more padding and rounded corner please. Try 2-3 dp rounded corner

Please update the screenshots

File: app/src/main/java/org/fossasia/openevent/general/utils/DataBindingAdapters.kt [hunk 1]
```diff
@@ -73,24 +75,39 @@ fun setExpired(imageView: ImageView, isExpired: Boolean) {
...
         }
     }
 }
+
+@BindingAdapter("ticketNumber")
+fun setTicketNumber(textView: TextView, ticketNumber: Int) {
+    if (ticketNumber == 1)
+        textView.text = "See 1 ticket"
+    else
+        textView.text = "See $ticketNumber tickets"
```
Comment (anhanh11001) [inline app/src/main/java/org/fossasia/openevent/general/utils/DataBindingAdapters.kt]: @iamareebjamal in here, when I use resources.getQuantityString from Resource file, it always return something like this "[Ljava.lang.Object;@88553da ticket". What should I do then or should I keep it this way?

Comment (iamareebjamal) [reply app/src/main/java/org/fossasia/openevent/general/utils/DataBindingAdapters.kt]: For now make it ticket(s)

**Comment by liveHarshit** [comment]
I don't see any order status in the provided screenshot?

File: app/src/main/java/org/fossasia/openevent/general/data/Resource.kt [hunk 2]
```diff
@@ -9,7 +10,13 @@ class Resource {
...
 
+    private val resources by lazy {
+        OpenEventGeneral.resource
+    }
+
     fun getString(@StringRes resId: Int) = context?.getString(resId)
 
     fun getString(@StringRes resId: Int, vararg args: Any?) = context?.getString(resId, args)
+
+    fun getColor(@ColorRes resId: Int) = resources?.getColor(resId)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/data/Resource.kt]: Just use context.resource

**Comment by anhanh11001** [comment]
> I don't see any order status in the provided screenshot?

Sorry I added the wrong screenshot

**Comment by anhanh11001** [comment]
updated

**Pull request merged**